### PR TITLE
Balance Changes

### PR DESCRIPTION
**Pull request by BrownDairy**

### CODE REVIEW

File: cardlibrary/ancientstirrings.lua [hunk 1]
```diff
@@ -1621,14 +1621,14 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
...
-		["Power"] = 600,
+		["Power"] = 800,
 		["Rarity"] = "Epic",
 		["AttackEffect"] = "Dash",
 		["Color"] = "Red", 
 		["Cost"] = {["Red"] = 2, ["Neutral"] = 1,},
 		["Effect"] = {
 			Name = "Pyrotechnics",
-			Description = "Generate a green icon.",
+			Description = "Generate two green iconw.",
```
Comment (Schrikvis) [inline cardlibrary/ancientstirrings.lua]: Please fix this typo

File: cardlibrary/ancientstirrings.lua [hunk 2]
```diff
@@ -1838,7 +1838,7 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 	
 	["Lezus 6"] = {
 		["Id"] = 662115933,
-		["Name"] = "Lezus",
+		["Name"] = "Token",
```
Comment (Schrikvis) [inline cardlibrary/ancientstirrings.lua]: Why is its name changed to Token?~ Shoudln't its rarity be changed to Token instead?

File: cardlibrary/ancientstirrings.lua (hunk 1 above)
Comment (Schrikvis) [inline cardlibrary/ancientstirrings.lua]: Please fix this typo

File: cardlibrary/ancientstirrings.lua (hunk 2 above)
Comment (BrownDairy) [inline cardlibrary/ancientstirrings.lua]: mistake on my part
will fix

**Pull request merged**